NetProbe/Model/NetworkMeasurement.mm

```diff
@@ -263,3 +288,146 @@ -(void) run {
 }
 
 @end
+
+@implementation WebConnectivity : NetworkMeasurement
+
+-(id) init {
+    self = [super init];
+    self.name = @"web_connectivity";
+    return self;
+}
+
+-(void) run {
+    self.test_id = [NSNumber numberWithDouble:[[NSDate date] timeIntervalSince1970]];
+    self.json_file = [NSString stringWithFormat:@"test-%@.json", self.test_id];
+    self.log_file = [NSString stringWithFormat:@"test-%@.log", self.test_id];
+    [TestStorage add_test:self];
+    setup_idempotent();
+    NSBundle *bundle = [NSBundle mainBundle];
+    NSString *path = [bundle pathForResource:@"urls" ofType:@"txt"];
+    
+    mk::ooni::WebConnectivity()
+    .set_options("backend", "https://a.collector.test.ooni.io:4444")
+    .set_options("nameserver", get_dns_server())
```

Comment:
Note to self: the existence of "nameserver" vs "dns/nameserver" is confusing. I should probably use a better name in this context to avoid the confusion.